conceptnet5/vectors/debias.py

```diff
@@ -390,14 +392,17 @@ def reject_subspace(frame, vecs):
     its rows have any correlation with any rows of `vecs`, by subtracting
     the outer product of `frame` with each normalized row of `vecs`.
     """
-    current_array = frame.copy()
+    current_array = frame.copy().values
     for vec in vecs:
         if not np.isnan(vec).any():
             vec = normalize_vec(vec)
             projection = current_array.dot(vec)
-            current_array -= np.outer(projection, vec)
+            np.subtract(current_array, np.outer(projection, vec), out=current_array)
 
-    return l2_normalize_rows(current_array, offset=1e-9)
+    current_array = pd.DataFrame(current_array, index=frame.index)
+    normalize(current_array.values, norm='l2', copy=False)
```

Comment:
I'm not quite sure why, but when I ran it, `normalize(current_array.values, norm='l2', copy=False)` didn't actually modify `current_array`. Interestingly, it seems like `normalize(result.values, norm='l2', copy=False)` in line 498 actually modifies `result`.